install.md

```diff
@@ -95,31 +95,9 @@ system](https://bodhi.fedoraproject.org/updates/?packages=podman).
 
 **Required**
 
-Fedora, CentOS, RHEL, and related distributions:
-
-```bash
-sudo yum install -y \
-  atomic-registries \
-  btrfs-progs-devel \
-  containernetworking-cni \
-  device-mapper-devel \
-  git \
-  glib2-devel \
-  glibc-devel \
-  glibc-static \
-  go \
-  golang-github-cpuguy83-go-md2man \
-  gpgme-devel \
-  iptables \
-  libassuan-devel \
-  libgpg-error-devel \
-  libseccomp-devel \
-  libselinux-devel \
-  make \
-  pkgconfig \
-  runc \
-  containers-common
-```
+Fedora, CentOS, RHEL, and related distributions you should try to run `make rpm`
+which will install dependencies, build the source and produce a rpm for the
+current platform.
 
```

Comment:
Having `make rpm` is cool and should be documented here. However, it is IMO not a good enough reason to delete the explicit dependencies from the install instructions.

Reply:
Is not that I want to delete them, is more that I want to avoid duplication. If they are duplicated they will get out of sync quickly, I can guarantee you.

Isn't acceptable to have a documented `make install-deps` target that does this instead?

I am also replacing `make rpm` with `make package` so it could be more platform-agnostic (and we can add support for deb,...). I am also removing the install and have separated in `make package-install` as user may not want to also install the package when building it.

Please feel free to look again get send feedback. It is very useful at this stage as I can easily change direction.

Reply:
> Is not that I want to delete them, is more that I want to avoid duplication. If they are duplicated they will get out of sync quickly, I can guarantee you.

Fair enough. Then it should be ideally split up into something like you're suggesting - installing dependencies, building the package (and potentially separately: installing it), ideally cross-platform. And those commands should be mentioned separately in the INSTALL-page.

Reply:
The jobs testing the rpm building on 3 platforms can be seen at https://review.rdoproject.org/r/#/c/23924/ -- once I get them green, I will start splitting this mega change in pieces that can be reviewed and merged. 

I already merged few sping-offs this week but more are needed.